### DISCUSSION TITLE
#Usability Issue.. even for single item edit is it is showing as Batch Edit. Due to this each edit field red marked needs to enable to edit

### DISCUSSION THREAD

**Discussion by Jeyakumarrathnaselvi** [directus/directus#13053] [Feature Requests]
#Usability Issue.. even for single item edit is it is showing as Batch Edit. Due to this each edit field red marked needs to enable to edit

### To Reproduce

Select one item in a tabular view
Click edit
It shows as Batch Edit
But it is not Batch edit due to this all editable fields needs to be enabled to edit which kills usability 

<details>
<summary>More info</summary>

### Errors Shown

Shown as batch edit for single item edit also

### What version of Directus are you using?

9.7.1

### What version of Node.js are you using?

16.14.2

### What database are you using?

Postgres

### What browser are you using?

Chrome

### How are you deploying Directus?

Through directus cloud

</summary>

**Comment by Jeyakumarrathnaselvi** [comment]
#Usability Issue.. even for single item edit is it is showing as Batch Edit. Due to this each edit field red marked needs to enable to edit

![image](https://user-images.githubusercontent.com/36895131/165879491-c19764e4-3062-455e-9af3-7c5cb7f96d1f.png)

**Comment by azrikahar** [comment]
Good point 👍 Technically it used to only be available when editing 2 or more items, hence the batch editing title/UX is in a sense working as intended, but it has since be allowed even for 1 item via #11318 which is also a great enhancement in and of itself. 

Definitely can be improved upon in terms of usability as stated. Maybe having the spreadsheet layout will alleviate this as well (thinking out loud here). I'll move this as a feature request as it's not exactly a bug pre se.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
